app/static/js/app.js

```diff
@@ -346,6 +347,12 @@ document
 document.getElementById("cancel-shutdown").addEventListener("click", () => {
   hideElementById("shutdown-confirmation-panel");
 });
+document.getElementById("shutdown-confirmation-panel").addEventListener("click", (e) => {
```

Comment:
Can we rename `e` to `evt` for consistency with other functions that use this convention?